client/src/assets/icons/index.tsx

```diff
@@ -0,0 +1,4 @@
+export {default as worldLogo} from './wordIcon.svg'
+export {RiFingerprintFill} from 'react-icons/ri'

```

Comment:
It seems you didn't run prettier on this code. The entire code is not formatted properly.

Before pushing, always run `yarn format:write`

Reply:
ok, i will do that now